### ISSUE TITLE
Interval not taken in account

### ISSUE THREAD

**Issue by TheSegfault** [wbhob/ionic-long-press#2]
Hi Wilson,

Interval is staying at 300ms even if I [interval]="2000"

Wait, I've been reading ur plugin, the interval is actually for "onPressing".

I thought interval was delaying the onPressStart

Can you validate ?

**Comment by wbhob** [comment]
interval is designed to fire on repeated press events. A way to address this would be like this:

Template:
```html
<button [interval]="2000" (onPressing)="onPress()" (onPressEnd)="release()"></button>
```

```ts
class Component {
    pressed = false;
    onPress() {
        if (!this.pressed) {
            this.pressed = true;
            // ... your implementation
        }
    }
    release() {
        this.pressed = false; 
    }
}
```

**Comment by TheSegfault** [comment]
Thanks for ur answer, this is what I did already :) I just got confused because of the "Ionic Long Press" name. I thought it was overloading ionic gesture "press" event to add delay to it.

**Comment by wbhob** [comment]
I may add this later. Leaving open for now

**Comment by jcphlux** [comment]
So I think this is not working correctly because you need to pass "this" ref into the run function.
```
this.zone.run(() => { }, this );
```
I will test this out in the PR I am working on for the other bug. If it resolved this issue I will include it.

**Comment by wbhob** [comment]
Awesome, thanks for your help!

**Comment by jcphlux** [comment]
When will you have time to review my PR? I am needing the NPM package updated for this soon.

**Comment by wbhob** [comment]
Fixed by #6
